components/tidb_query/src/lib.rs

```diff
@@ -11,6 +11,11 @@
 #![feature(specialization)]
 #![feature(const_fn)]
 #![feature(test)]
+// FIXME: rustc says there are redundant semicolons here but isn't
+// saying where as of nightly-2019-08-26
+#![allow(redundant_semicolon)]
```

Comment:
Do we still need this allow?

Reply:
@brson

Reply:
I'll upgrade the toolchain again, re-check this, and add a link to the upstream bug.

Reply:
It's still needed.